flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -237,53 +236,26 @@ public ExecutionConfig setParallelism(int parallelism) {
 	}
 
 	/**
-	 * Gets the number of times the system will try to re-execute failed tasks. A value
-	 * of {@code -1} indicates that the system default value (as defined in the configuration)
-	 * should be used.
+	 * Sets the restart strategy configuration which defines which restart strategy shall be used
+	 * for the execution graph of the corresponding job.
```

Comment:
Agreed. Good point. I've simplified the the description and added a code example.